.linthtmlrc.json

```diff
@@ -1,3 +1,49 @@
 {
```

Comment:
I was unable to launch the project locally so I couldn't include styling issues. 
With current package-lock I was unable to install dependencies plus I got many, many deprecation errors.
Removing old package-lock helped to install dependencies, but then I got this error page:
![obraz](https://user-images.githubusercontent.com/13736421/220347993-62bc1461-70ef-459a-be63-a7d04bf76980.png)